### ISSUE TITLE
Add the ability to import from /etc/apt/sources.list.d/ so we can import all sources.

### ISSUE THREAD

**Issue by mwoolweaver** [wstyres/Zebra#1065]
**Is your feature request related to a problem? Please describe.**

Not a problem more of a caveat I think. 

If you install [jonluca/MasterRepo](https://github.com/jonluca/MasterRepo) w/ Cydia and then attempt to import those Cydia sources via Zebra it won't work as Zebra is [coded](https://github.com/wstyres/Zebra/blob/54c6de465ce0208f2fd9a2703a3f2f4d35fbab5a/Zebra/Tabs/Repos/Helpers/ZBRepoManager.m#L481) to only look for files provided by the package managers (Cydia, Installer, Sileo, etc.).

**Describe the solution you'd like**

Add the ability to import from ```/etc/apt/sources.list.d/``` w/o using a separate app. So we can import all Cydia sources.

**Describe alternatives you've considered**

using a separate app (Filza, iFile, etc.) and share sheet actions.

**Additional context**

Also worth noting if you install that same package w/ Zebra the sources will not be added to Zebra's source list in the app.

**Comment by wstyres** [comment]
Zebra is configured to use its own sources.list which resides in Zebra's documents directory and doesn't read any sources from `sources.list.d/` so that is why the MasterRepo install package does not work on Zebra, the package would have to be configured to either append to Zebra's sources.list or install to Zebra's directory (which I would have to add support for reading from an entire directory).

That's beside the request here though, just thought that I would provide that information for you as well. UI wise this is a bit tricky. My first thought is just to import all files with .list in their names from `sources.list.d/` when importing from Cydia and the same situation with .sources for Sileo. I'm not sure if Sileo is able to read .list files though so it would require some testing.

Basically what I'm thinking is to have all those files in the directory import when trying to import from Cydia so that the MasterRepo lists can be imported properly as well without an obscure "Import from sources.list.d" option.

**Comment by mwoolweaver** [comment]
> Basically what I'm thinking is to have all those files in the directory import when trying to import from Cydia


That would definitely fix the issue

**Comment by wstyres** [comment]
Alright then, I'll do that.

**Comment by wstyres** [comment]
Implemented in c82f6d8a7b2dc36dc83e81490f48751d84800b15. "Import from Cydia" will now import any .list files found in /etc/apt/sources.list.d and "Import from Sileo" will now import and .sources files found in /etc/apt/sources.list.d (or /etc/apt/sileo.list.d on checkra1n).
